ports/operationsteel/Operation S.T.E.E.L.sh

```diff
@@ -22,14 +22,19 @@ get_controls
 GAMEDIR=/$directory/ports/operationsteel/
 CONFDIR="$GAMEDIR/conf/"
 
-# Ensure the conf directory exists
+> "$GAMEDIR/log.txt" && exec > >(tee "$GAMEDIR/log.txt") 2>&1
+
 mkdir -p "$GAMEDIR/conf"
+cd $GAMEDIR
+
+#  If XDG Path does not work
+#$ESUDO rm -rf ~/.operationsteel
```

Comment:
`#` means its commented out so not active.
This meant if the game does use it uncomment else don#t use it.